main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -68,9 +69,9 @@ sealed trait Kind {
 object Kind {
 
   /**
-    * Represents a kind variable.
+    * Represents a wild kind.
```

Comment:
Lets a description: It is a kind that has not yet been determined, but will be determined and replaced by the Kinder.

Reply:
Right?

Reply:
I think some Wild kinds can escape to later phases, but only if they are somehow irrelevant. E.g. if we have (`def f(x: m[_]): Int = ???`, the underscore might have kind `Wild`.